### DISCUSSION TITLE
MapLibre not working on VSC (1/2)

### DISCUSSION THREAD

**Discussion by mmoranv** [giswqs/geog-312#157] [Q&A]
leafmap maps work on Visual Code. leafmap maps that include anything related to maplibre don't, whether I need an API key or not. I looked at other responses, and created environments only loading leafmap and maplibre as suggested, virtual environments too, and tnothing works. No error pops up, it simply doesn't show anything. 
![Screenshot 2024-11-27 143818](https://github.com/user-attachments/assets/02cf1970-96d2-4525-8b75-f07316da7e08)

**Comment by giswqs** [comment]
This is a known issue. See https://github.com/eoda-dev/py-maplibregl/issues/113
